lib/bolt/transport/orch.rb

```diff
@@ -22,6 +22,8 @@ def self.options
         %w[service-url cacert token-file task-environment local-validation]
       end
 
+      PROVIDED_FEATURES = ['puppet-agent'].freeze
```

Comment:
I think we probably *should* do validation of whether there is a suitable implementation if local-validation is true.

I'm not sure how useful that actually is though

Reply:
There's no way we *could* make use of it, other than for local validation. This can't work with orchestrator until orchestrator supports it.